docs/reference/announcements-release-notes/overview.md

```diff
@@ -0,0 +1,65 @@
+| 8.5     | 9 April 2024    | [Release notes](/reference/announcements-release-notes/850/850-release-notes.md) | [Announcements](/reference/announcements-release-notes/850/850-announcements.md)            | [Update 8.4 to 8.5](/self-managed/operational-guides/update-guide/840-to-850.md) | 14 October 2025              | - [ Camunda 8 core ](https://github.com/camunda/camunda/releases/tag/8.5.0) <br /> - [ Connectors ](https://github.com/camunda/connectors/releases/tag/8.5.0)            | [Release blog](https://camunda.com/blog/2024/04/camunda-8-5-release/)                                            |
+| 8.4     | 9 January 2024  | -                                                                                | [Announcements](/reference/announcements-release-notes/850/850-announcements.md#camunda-84) | [Update 8.3 to 8.4](/self-managed/operational-guides/update-guide/830-to-840.md) | 9 July 2025                  | - [ Camunda Platform ](https://github.com/camunda/camunda-platform/releases/tag/8.4.0) <br /> - [ Connectors ](https://github.com/camunda/connectors/releases/tag/8.4.0) | [Release blog](https://camunda.com/blog/2024/01/camunda-8-4-simplifying-installation-enhancing-user-experience/) |
+| 8.3     | 10 October 2023 | -                                                                                | [Announcements](/reference/announcements-release-notes/850/850-announcements.md#camunda-83) | [Update 8.2 to 8.3](/self-managed/operational-guides/update-guide/820-to-830.md) | 9 April 2025                 | - [ Camunda Platform ](https://github.com/camunda/camunda-platform/releases/tag/8.3.0) <br /> - [ Connectors ](https://github.com/camunda/connectors/releases/tag/8.3.0) | [Release blog](https://camunda.com/blog/2023/10/camunda-8-3-scaling-automation-maximize-value/)                  |
+
+:::note
+Camunda Cloud 1.3 is available in the site dropdown, but is unsupported. However, the corresponding Optimize version for Camunda Cloud 1.3, Optimize 3.7, remains supported at this time.
+:::
+
+### Changelogs
+
+Technical changelogs for Camunda 8 are available on GitHub:
+
+- [Camunda 8 Core](https://github.com/camunda/camunda/releases)
+- [Connectors](https://github.com/camunda/connectors/releases)
+- [Camunda Helm chart](https://github.com/camunda/camunda-platform-helm/releases)
+
+Most releases and changelogs are available through the [Camunda Platform repo](https://github.com/camunda/camunda-platform). The following component release notes are available as linked below:
+
+- [Desktop Modeler](https://github.com/camunda/camunda-modeler/releases)
+- [Connectors](https://github.com/camunda/connectors/releases)
+
+Some components have changelogs that are not publicly available, including the following:
+
+- Web Modeler
+- Console
+
+These are available in a blog format under the [release notes](https://camunda.com/blog/category/releases/) tag.
```

Comment:
I propose:
* Let's just not talk about the fact that web modeler has no changelog publicly
* camunda-platform repo is not maintained beyond 8.6+ (cc. @hisImminence ). I would not reference it.

```suggestion
Technical changelogs for Camunda 8 are available on GitHub:

- [Camunda 8 Core](https://github.com/camunda/camunda/releases)
- [Connectors](https://github.com/camunda/connectors/releases)
- [Camunda Helm chart](https://github.com/camunda/camunda-platform-helm/releases)
- [Desktop Modeler](https://github.com/camunda/camunda-modeler/releases)

```

Reply:
Would it make sense to just add "Web Modeler and Console updates are available in a blog format under the [release notes](https://camunda.com/blog/category/releases/) tag" or is this no longer accurate?